src/Altinn.Broker.Application/ConfigureResource/ConfigureResourceHandler.cs

```diff
@@ -66,6 +66,22 @@ public async Task<OneOf<Task, Error>> Process(ConfigureResourceRequest request,
                 return updateManifestFileShimResult.AsT1;
             }
         }
+        if (request.ExternalServiceCodeLegacy is not null && request.UseManifestFileShim == true)
```

Comment:
Consider changing this to when UseManifestFileShim == true, it is required that both ExternalServiceCodeLegacy and ExternalServiceEditionCodeLegacy is set, so that they are always available for Legacy / Manifest file useage.

Reply:
Good catch

If they are dependent on eachother, we should probably make a better validation, to give feedback to the user. 


Otherwise, if they are not dependent, but should be able to update independently:
If request.UseManifestFileShim is null(meaning do not update this), you cant update the new variables. 
So if request.useManifestFileShim is null, you have to check what the current value is.